ops/ai-eng/contracts-test-maintenance/prompt/prompt.md

```diff
@@ -85,16 +85,16 @@ This systematic approach ensures comprehensive test improvements without missing
 **Phase 3 - Implementation & Validation**
 *Goal: Apply improvements while maintaining all tests passing*
 - Implement enhancements identified in Phase 1
-- Commit changes using conventional format
```

Comment:
I would keep this line. It's not contradicting the additions here. It's referring to https://www.conventionalcommits.org/en/v1.0.0/